test/scripts/bootstrap.sh

```
@@ -66,6 +66,7 @@ if [ "$ENABLE_TRAFFIC_MANAGER" == "false" ]; then
        --resource-group $RESOURCE_GROUP \
        --name $HUB_CLUSTER \
        --node-count $NODE_COUNT \
+       --node-vm-size Standard_A2_v2 \
```

where is the fix for fail silently bug when 'yq' is  missing?